core/src/test/scala/unit/kafka/utils/PasswordEncoderTest.scala

```diff
@@ -109,11 +109,9 @@ class PasswordEncoderTest {
     verifyEncodeDecode(keyFactoryAlg = None, "AES/CFB/PKCS5Padding", keyLength = 128)
```

Comment:
nit: unused import

Reply:
Thanks for catching this. I removed this line before merging the PR.